keras/backend/tensorflow/nn.py

```diff
@@ -802,3 +802,62 @@ def ctc_loss(
         blank_index=mask_index,
         logits_time_major=False,
     )
+
+
+def ctc_decode(
+    inputs,
+    sequence_length,
+    strategy,
+    beam_width=100,
+    top_paths=1,
+    merge_repeated=True,
+    mask_index=None,
+):
+    """Decodes the output of a softmax using CTC (Connectionist Temporal
+    Classification).
+
+    Arguments:
+        inputs: Tensor `(batch_size, max_length, num_classes)` containing the
+            output of the softmax.
+        sequence_length: Tensor `(batch_size,)` containing the sequence length
+            for each output sequence in the batch.
+        strategy: String for decoding strategy to use. Supported values
+            are "greedy" and "beam_search".
+        beam_width: Integer specifying the beam width to use for beam
+            search strategies. Ignored if `strategy` is "greedy".
+        top_paths: Integer specifying the number of top paths to return.
+            Ignored if `strategy` is "greedy".
+        merge_repeated: Boolean specifying whether to merge repeated labels
+            during decoding.
+        mask_index: Integer specifying the index of the blank label.
+
+    Returns:
+        A tuple of a list of `SparseTensor` containing the decoded sequences
+        and a list of `Tensor` containing the negative of the sum of probability
+        logits (if strategy is `"greedy"`) or the log probability (if strategy
+        is `"beam_search"`) for each sequence.
+    """
+    inputs = tf.convert_to_tensor(inputs)
+    inputs = tf.transpose(inputs, (1, 0, 2))
+
+    sequence_length = tf.convert_to_tensor(sequence_length)
+    if strategy == "greedy":
+        decoded, probs = tf.nn.ctc_greedy_decoder(
```

Comment:
You can just do `return tf.nn.ctc_greedy_decoder(...`

Reply:
This is done, thanks for noticing.